### PR TITLE
ENH: adds authorship guidelines

### PR DESCRIPTION
**Pull request by deniederhut**
Closes https://github.com/scipy-conference/scipy_proceedings/issues/375

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +172,8 @@ have final say in whether to accept or reject a paper.
 - For your paper to be found by the build system at http://procbuild.scipy.org
   your PR needs to have a title that begins with "Paper:". If you do not do
   this, the co-chairs will change your title on your behalf.
+- Authors may include a project or consortium (e.g. [The Jupyter Project](https://raw.githubusercontent.com/scipy-conference/scipy_proceedings/2018/papers/project_jupyter/paper.rst))
+- There must be at least one corresponding author, and this must be a specific person
```
Comment (stargaser) [inline README.md]: perhaps add "with a valid email address"?

File: README.md [hunk 2]
```diff
@@ -172,6 +172,8 @@ have final say in whether to accept or reject a paper.
 - For your paper to be found by the build system at http://procbuild.scipy.org
   your PR needs to have a title that begins with "Paper:". If you do not do
   this, the co-chairs will change your title on your behalf.
+- Authors may include a project or consortium (e.g. [The Jupyter Project](https://raw.githubusercontent.com/scipy-conference/scipy_proceedings/2018/papers/project_jupyter/paper.rst))
```
Comment (stargaser) [inline README.md:175]: This is a good idea. I feel like these will be footnotes, and that perhaps we should say so? Unless this is intended to be something else, like an extra affiliation for each author.

**Review by stargaser — COMMENTED** [review]
These seem like good additions. I'm wonder if we need to add a little more detail, per the inline comments.

File: README.md (hunk 2 above)
Comment (deniederhut) [reply README.md:175]: The idea is to be able to list a consortium as an author. I believe Jupyter, for example, lists the project as first author.

File: README.md (hunk 1 above)
Comment (deniederhut) [reply README.md]: Done!

**Pull request merged**